app/views/pages/faq.html.haml

```diff
@@ -88,7 +88,7 @@
   %p
     Tweet or share Password Pusher.
   %a{ :target => "_blank", :href => "http://ctt.ec/dapfb", :alt => "Tweet: PasswordPusher - A better way to share passwords than email #sysadmin #technology #netsec http://ctt.ec/dapfb+" }
```

Comment:
Could you also change this URL to https?  http://ctt.ec/dapfb  Tested and it works.